packages/gestalt/src/ModuleExpandable.js

```diff
@@ -17,15 +18,22 @@ type Props = {|
     type?: 'error' | 'info',
     children?: Node,
   |}>,
+  onExpandedChange?: (?number) => void,
 |};
 
 export default function ModuleExpandable({
   id,
   accessibilityExpandLabel,
   accessibilityCollapseLabel,
+  expandedIndex,
+  onExpandedChange,
   items,
 }: Props): Node {
-  const [expandedId, setExpandedId] = useState(-1);
+  const [expandedId, setExpandedId] = useState(expandedIndex || null);
```

Comment:
Since expandedIndex can be 0 (a falsy value), this needs to be more exact:
```js
useState<?number>(Number.isFinite(expandedIndex) ? expandedIndex : null)
```